python/batch-predictor/docker/app.Dockerfile

```diff
@@ -15,23 +15,50 @@
 ARG BASE_IMAGE
 FROM ${BASE_IMAGE}
 
+ARG MLFLOW_ARTIFACT_STORAGE_TYPE
+
 ARG GOOGLE_APPLICATION_CREDENTIALS
-RUN if [ ! -z "${GOOGLE_APPLICATION_CREDENTIALS}" ]; \
-    then gcloud auth activate-service-account --key-file=${GOOGLE_APPLICATION_CREDENTIALS} > gcloud-auth.txt; \
-    else echo "GOOGLE_APPLICATION_CREDENTIALS is not set. Skipping gcloud auth command." > gcloud-auth.txt; \
+
+ARG AWS_ACCESS_KEY_ID
+ARG AWS_SECRET_ACCESS_KEY
+ARG AWS_DEFAULT_REGION
+ARG AWS_ENDPOINT_URL
+
+RUN if [ "${MLFLOW_ARTIFACT_STORAGE_TYPE}" = "gcs" ]; then  \
```

Comment:
would it better to separate dockerfile based on cloud storage or cloud provider?

Reply:
Yeah I was thinking of that too, but that'll require some additional rework of our CICD pipelines + publishing new images.. but yeah I can do that.

Reply:
For this part, we've decided to work on this another time as an enhancement as it requires more extensive reworking of the existing CICD pipelines as well as Merlin config files to work. I'll create another card in our backlog to keep track of this.